app/scripts/models/auth_brokers/base.js

```diff
@@ -271,6 +271,10 @@ define([
        */
       emailVerificationMarketingSnippet: true,
       /**
+       * Should the view try to synchronise sign-in state across tabs?
+       */
+      interTabSignIn: true,
```

Comment:
Something about the name doesn't seem quit right, but I don't have a definitively better name.